content/Solutions/Optimizations/Security.md

```diff
@@ -44,7 +60,7 @@ Leaving the **Authorized Networks** or **Authorized Hosts and IP addresses* list
 To define which systems can connect to the share, click the **Advanced Options** and enter all networks, hosts, and IP addresses to have share access.
 All other systems are denied access.
 
-## SMB
+### SMB
 
 Using Server Message Block (SMB) to share data is a very common situation for TrueNAS users.
 However, it allows outside connections to the system and must be properly use to avoid security concerns.
```

Comment:
incorrect use of italics in line 74, 77, 80, 83, and 85, all the UI elements should be bold